### PR TITLE
Refactor shellscripts

### PR DESCRIPTION
**Pull request by Mic92**

### CODE REVIEW

**Comment by estesp** [comment]
ping @tianon

**Comment by calavera** [comment]
I'm not a bash expert but the changes look consistent to me.

File: hack/make/test-integration-cli [hunk 1]
```diff
@@ -9,23 +9,24 @@ bundle_test_integration_cli() {
 
 # subshell so that we can export PATH without breaking other things
 (
-	source "$(dirname "$BASH_SOURCE")/.integration-daemon-start"
+	SCRIPTDIR=$(dirname "$BASH_SOURCE")
```
Comment (tianon) [inline hack/make/test-integration-cli]: This script can just assume `SCRIPTDIR` comes down from `make.sh`, right?

Comment (tianon) [inline hack/make/test-integration-cli]: (and have a full path to `.integration-daemon-start` instead of this relative one?)

Comment (tianon) [inline hack/make/test-integration-cli]: Just thinking that would make sure our usage of what `SCRIPTDIR` means stays consistent to be referring to the `hack` directory.

File: hack/release.sh [hunk 2]
```diff
@@ -71,23 +71,23 @@ BUCKET=$AWS_S3_BUCKET
...
 }
 
 # write_to_s3 uploads the contents of standard input to the specified S3 url.
 write_to_s3() {
 	DEST=$1
 	F=`mktemp`
-	cat > $F
-	s3cmd --acl-public --mime-type='text/plain' put $F $DEST
-	rm -f $F
+	echo > "$F"
```
Comment (tianon) [inline hack/release.sh]: This one was `cat` on purpose -- this function takes stdin and puts it into the file at `$F`.

Comment (Mic92) [inline hack/release.sh]: Sorry for breaking, fixed.

**Comment by thaJeztah** [comment]
ping @tianon PR was updated (commits may need to be squashed)

**Comment by tianon** [comment]
LGTM cc @jfrazelle

**Comment by jfrazelle** [comment]
LGTM

**Pull request merged**